app/presenters/outcome_presenter.rb

```diff
@@ -1,7 +1,25 @@
 require 'erubis'
 
 class OutcomePresenter < NodePresenter
-  class OutcomeTemplateMissing < StandardError; end
+  class ViewContext
+    def initialize(state)
+      @state = state
+      state_methods = state.methods - SmartAnswer::State.new(nil).methods
+      @state_reader_methods = state_methods.reject { |m| m =~ /=$/ }
+    end
+
+    def method_missing(method, *args, &block)
```

Comment:
I'm not sure I can. I need to inspect the `@state` object to determine which methods to delegate to it and I'm currently doing that in the initializer. Unless I'm missing a different way of using `delegate`?